README.md

```diff
@@ -10,28 +10,31 @@ It is part of [Membrane Multimedia Framework](https://membraneframework.org).
 
 ## Installation
 
-First, you need to install FFmpeg on your system:
+Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 
-### macOS
-
-```shell
-brew install ffmpeg
+```elixir
+{:membrane_camera_capture_plugin, "~> 0.6.0"}
```

Comment:
I know it's outside of the scope of this PR, but can you fix that weird tab here?

Reply:
Do you mean lack of a tab in this elixir section?